superset-frontend/src/components/Alert/index.tsx

```diff
@@ -60,12 +63,13 @@ export default function Alert(props: AlertProps) {
       icon={<AlertIcon aria-label={`${type} icon`} />}
       closeText={closable && <Icon name="x-small" aria-label="close icon" />}
       css={{
-        padding: '6px 10px',
+        marginBottom: roomBelow ? gridUnit * 4 : 0,
+        padding: `${gridUnit * 2}px ${gridUnit * 3}px`,
         alignItems: 'flex-start',
         border: 0,
         backgroundColor: baseColor.light2,
         '& .ant-alert-icon': {
-          marginRight: 10,
+          marginRight: gridUnit * 2,
```

Comment:
Happened to see some pixel measurements in here that I figured I might as well convert to gridUnits while we're in here. Trying to keep bycatch minimal these days, but... I couldn't help myself.